react/util/ui-utils.js

```diff
@@ -144,6 +146,14 @@ export const notify = function (message) {
         : Alert.alert(message);
 };
 
+export const toClipboard = function (value, notification = true) {

```

Comment:
We already had a copy of fields to the clipboard somewhere. Where was it?

Reply:
we [have in ripe-robin](https://github.com/ripe-tech/ripe-robin-revamp/blob/master/components/info-user/info-user.js#L100) but [not here](https://github.com/ripe-tech/ripe-components-react-native/search?q=clipboard)